File: app/src/main/java/com/alpha/acamera/camera/CameraInfo.java
package com.alpha.acamera.camera;

import android.hardware.Camera;

/**
 * 相机信息类
 */
public class CameraInfo {
    private static final String TAG = "CameraManager";

    static public int HEAD_CAMERA = Camera.CameraInfo.CAMERA_FACING_FRONT;//前置摄像
    static public int BACK_CAMERA = Camera.CameraInfo.CAMERA_FACING_BACK;//后置摄像
}

File: app/src/main/java/com/alpha/acamera/camera/CameraInfo.kt
package com.alpha.acamera.camera

import android.hardware.Camera

/**
 * 相机信息类
 */
object CameraInfo {
    private const val TAG = "CameraManager"
    var HEAD_CAMERA = Camera.CameraInfo.CAMERA_FACING_FRONT //前置摄像
    var BACK_CAMERA = Camera.CameraInfo.CAMERA_FACING_BACK //后置摄像
}